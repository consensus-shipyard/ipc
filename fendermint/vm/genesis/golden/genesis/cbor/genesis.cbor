a76a636861696e5f6e616d65606974696d657374616d701bb7b9f4a2f96c8d266f6e6574776f726b5f76657273696f6e1268626173655f666565587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff409ae7fc909ef033e6fc9a79a4b297ee3e8599fb8a23f31302ec4047919127eeffffffffffffffff2f86056bb0bfd5768545831477c093946b706f7765725f7363616c65036a76616c696461746f727387a26a7075626c69635f6b65795841048a543f8064fc77f3d37566b9cb138629555ba0b134e376a36130e8cbba6b485df82c022b59e5b26b93c086d73293e95d8bbff1537c52911abdd773dc9b82761465706f77657258390028d2b2f3e9037cf2ffffffffffffffffffffffffffffffff537df730d626c0018171798a46e593e7af6bad846755d8530000000000000001a26a7075626c69635f6b657958410434e468899dac7be5edcc51905def584f560d2a2842ace904819cf7748efe37149b22ff7a10d28061a8758ab2341dbf9d413b460d7175261148edea25be06780a65706f776572587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff455ba0d741ec3f8affffffffffffffff34f9eb2c92d80dcaa1918930962fb1e3faa1ef6a6e586baca26a7075626c69635f6b657958410482413345f1b39ad7f620b7a0298df10cd8dfc2ba53119173e8170d3ea002201f4f6feb45967088d0ba10bfb03eae0a9bf6ebc149b58acbd57d7b13dfc4cb894e65706f776572587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff2a05e94d9793f9fdc65ce1e6b73933290000000000000000fc2e53b6d873e6e675c7517a620a3deefffffffffffffffe10ab95aba40c2119a26a7075626c69635f6b65795841047f520119fbcc082ae625d7185aad9cff7c7b609d270b77742b5d5773acf2c47726278df2d4465b474b845dfc7fd6b64f3ebd7a4ebedf802b7a87b77b46889fa765706f776572587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff2c030db1dcde54d6a26a7075626c69635f6b65795841049400208251a16c5ab4417cf9298f03dedd59d61c3cf98217aeae62aba6243155b5ef9efc75c26de713fc8d24621acaf6325cb5ce16bc70c74d6a9b4a23914bae65706f776572587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff6f4c667456b77975907a34ceb3429117484e7de48e57d7824ed2dc0660d546c4459935e8ffcb4f7aa26a7075626c69635f6b657958410433df8142a82e07463734dd3e197d6dc304d3434d79297defb7b203ff98e340a09d2a8b57256967129c009d1a76199cf10e13794f5e86a659bf92237b9f35a69165706f776572587d00fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffeea805ca325f02a52ea8c53c1c85bfaf6dc13891a2b74155fe08366ad62eb5444a26a7075626c69635f6b6579584104705a8aea5063c4bb4d9e4c667b4f8d07a98933a84098edb5f00915e227dfb03049e7f95d33b4ff53e816acacc53096e2fc4300f17acdd43e8eb19bd4a7b7028165706f776572582900fb291f790e3d8e00ccd99c9e7079fab1d421e35a4f523fb6a817ddc07fd874ba368ce00828f0930d686163636f756e747389a2646d657461a1674163636f756e74a1656f776e657256040a8927d4e9b75ee744313b105867c4ef616e5278cf6762616c616e6365587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff0000000000000000b2251727208b491a91f50acd8a9e97acad53514a3ed024fca2646d657461a1674163636f756e74a1656f776e6572550155ebcbf71f180775978b2b451a07994a42ed0d866762616c616e6365584100e66ed5940088df145c5f3306cabb8ca1f11bca6f2a74595001169f801d825d9362a62ff67e2e00c713aebcc757f05f318c84d4b4005c2db50000000000000000a2646d657461a1674163636f756e74a1656f776e6572550113c872d2d88a5527927f19b525de69f5961b9eb06762616c616e6365587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff3a1aceecb21631398f81efd37e0fd5e2d7418e096236aa47b7e4af8f28da13be3ddece5a4504d65da2646d657461a1684d756c7469736967a4677369676e657273835501db0ea0f6edb3d2b3f47aac1ed738c0ecf50732e555011084b4a7323aa8b3d0b08d18f0c7700b38f6610f55017d7bfba7861d98176844612bef55be3579ecdd63697468726573686f6c64017076657374696e675f6475726174696f6e1bf40d669ff38c16d76d76657374696e675f73746172741beced5b4d37032e566762616c616e636540a2646d657461a1674163636f756e74a1656f776e657256040a4ad3019399042811b434c85af4b52e0687da44ec6762616c616e6365587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff112bb3bba2d5d7f93435adbe1517519bbaf7cca2be3ae354b251e025980506c2a2646d657461a1684d756c7469736967a4677369676e65727384550114800de7536cfc2e82aa20e921409d77cbd0fbcd55012fc2bdff13bb5c2b4775ff2a53d0e367079690d355011460ecaa73b57c86abb3ed04d5874707232613935501e9ae45e4adc9e8c391700a319b5507b6f9d9e690697468726573686f6c64017076657374696e675f6475726174696f6e1bed5885aee2229e226d76657374696e675f7374617274006762616c616e63655839004d5100d6f86541f50000000000000001dfc9775f536a2524a2706ede7b594875a1badec301e45780fe854ea0d67bd61f8cd5a3bd7926ef61a2646d657461a1674163636f756e74a1656f776e657256040ab3be940f9a8100886b9b12c206432711a668cbf76762616c616e63655831009f350f8009e90a2c1bc0cdeec05c5e9a9d9f85b16c87303e15fce6f51119e223139f40175b279e182d883143ee3273d7a2646d657461a1674163636f756e74a1656f776e657255015c05d022dbce2f8f20b81b2d965fcc8ae6d056eb6762616c616e6365587d00ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff86d1bbe70f1b39b7e6af6503638691f3b1287db00cd7447f0000000000000000ffffffffffffffffdf054ca5d03dc4d8eb5bb7f2d4311c32f00c97125d128ca3aa56eef95e6c58e4a2646d657461a1674163636f756e74a1656f776e65725501792ab55ca2108894731edb9beb0b237fc762eb2e6762616c616e636540