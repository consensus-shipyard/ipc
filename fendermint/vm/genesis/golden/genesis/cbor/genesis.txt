Genesis { chain_name: "", timestamp: Timestamp(13238881560438803750), network_version: NetworkVersion(18), base_fee: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869404826996123942996680521032405130894660606662866425527020487412694275558041583972113209956456927736010875789202881814822.188575537313649556), power_scale: 3, validators: [Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [40585309, 3814126, 57292307, 46453645, 22371232, 46457226, 55995292, 31444813, 58746108, 2266383], magnitude: 1, normalized: true }, y: Field { n: [58881556, 31258406, 17935325, 21885258, 25935857, 10811991, 552307, 46771791, 36395493, 4066048], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(115905345978841588980225147314149826556621442596139603007579857206687410532487259944015461143575433701903051237968272.367388145240506369)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [50214676, 54385955, 43010073, 10554035, 55971114, 58447379, 12916997, 32479159, 9018796, 866586], magnitude: 1, normalized: true }, y: Field { n: [33978378, 58362223, 39916686, 3524052, 21052230, 17264615, 56142627, 33654433, 58331346, 2541759], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758522478052414553133115502494135655411742889367950963585048499153177857209974810104.011273695665875884)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [139295, 29577128, 18300545, 48843846, 14213058, 40074307, 34306562, 40591320, 54915507, 2134092], magnitude: 1, normalized: true }, y: Field { n: [13338958, 46462961, 12408791, 19322411, 66513857, 61571750, 17562371, 35865320, 54892144, 1301498], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869404762880966971366582949747317347562482300944928163297661574506337462696587718587360421631179433713880846551761356309790.292650627951239449)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [15910007, 55958763, 58147509, 41196589, 58489696, 44787519, 39678341, 2141080, 18480076, 2086016], magnitude: 1, normalized: true }, y: Field { n: [42508199, 32366289, 178088, 20642686, 54443386, 66432403, 4579271, 23928110, 32691270, 625123], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758524035337792904616958075776435777990406039363527010043736240963055342423554029877.788660629289981142)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [35926357, 60336873, 2194154, 7402470, 48060886, 40091895, 1560466, 28404433, 8540577, 2424840], magnitude: 1, normalized: true }, y: Field { n: [59853742, 44487304, 51147990, 54024945, 36854965, 8827581, 63492678, 28810319, 50099650, 2980839], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758522827991465549568102520293365829011494007770367260598451834327271514472485444898.093673655225241466)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [14893216, 8454118, 64945019, 20309157, 50647875, 39803760, 55432161, 1906908, 21145646, 849888], magnitude: 1, normalized: true }, y: Field { n: [53847697, 8969959, 40213497, 20806170, 17699705, 25585468, 643495, 27019888, 56042857, 2575010], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758524035337792904616957950260259655264547820235996513366173928456949670201941552993.858270223651263556)) }, Validator { public_key: ValidatorKey(PublicKey(Affine { x: Field { n: [64991280, 38107273, 47931136, 44106339, 61442355, 47440705, 31770215, 51572022, 48910435, 1840802], magnitude: 1, normalized: true }, y: Field { n: [62325377, 6747433, 21227755, 63302455, 50086656, 21767608, 23775948, 66932640, 22885300, 1210878], magnitude: 1, normalized: true }, infinity: false })), power: Collateral(TokenAmount(2095608841687035459301632016541676534649917040055276535546966886006214145827036.319530804197757709)) }], accounts: [Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Delegated(DelegatedAddress { namespace: 10, length: 20, buffer: [137, 39, 212, 233, 183, 94, 231, 68, 49, 59, 16, 88, 103, 196, 239, 97, 110, 82, 120, 207, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }) }), balance: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758524035337792904616957959984346540674210620160641860186130561031376451909822311478.51031784684467942) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Secp256k1([85, 235, 203, 247, 31, 24, 7, 117, 151, 139, 43, 69, 26, 7, 153, 74, 66, 237, 13, 134]) }) }), balance: TokenAmount(12068752682217344880722671766340107503739249550210705844171550551337242323526284799940761943090995477947529220758650143244107871320720338.344605255804649472) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Secp256k1([19, 200, 114, 210, 216, 138, 85, 39, 146, 127, 25, 181, 37, 222, 105, 245, 150, 27, 158, 176]) }) }), balance: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758522384159071328135781738404981702344468123526003070553355590735581825669842239531.300417032979535453) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address { payload: Secp256k1([219, 14, 160, 246, 237, 179, 210, 179, 244, 122, 172, 30, 215, 56, 192, 236, 245, 7, 50, 229]) }), SignerAddr(Address { payload: Secp256k1([16, 132, 180, 167, 50, 58, 168, 179, 208, 176, 141, 24, 240, 199, 112, 11, 56, 246, 97, 15]) }), SignerAddr(Address { payload: Secp256k1([125, 123, 251, 167, 134, 29, 152, 23, 104, 68, 97, 43, 239, 85, 190, 53, 121, 236, 221, 99]) })], threshold: 1, vesting_duration: 17585824957123532503, vesting_start: 17072402149624983126 }), balance: TokenAmount(0.0) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Delegated(DelegatedAddress { namespace: 10, length: 20, buffer: [74, 211, 1, 147, 153, 4, 40, 17, 180, 52, 200, 90, 244, 181, 46, 6, 135, 218, 68, 236, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }) }), balance: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872862722356017328064756151166307827869405370407152286801072676024887272960758524035337792904616957967750879861629873436340370888422120258980765650245697869754.214023408635938498) }, Actor { meta: Multisig(Multisig { signers: [SignerAddr(Address { payload: Secp256k1([20, 128, 13, 231, 83, 108, 252, 46, 130, 170, 32, 233, 33, 64, 157, 119, 203, 208, 251, 205]) }), SignerAddr(Address { payload: Secp256k1([47, 194, 189, 255, 19, 187, 92, 43, 71, 117, 255, 42, 83, 208, 227, 103, 7, 150, 144, 211]) }), SignerAddr(Address { payload: Secp256k1([20, 96, 236, 170, 115, 181, 124, 134, 171, 179, 237, 4, 213, 135, 71, 7, 35, 38, 19, 147]) }), SignerAddr(Address { payload: Secp256k1([233, 174, 69, 228, 173, 201, 232, 195, 145, 112, 10, 49, 155, 85, 7, 182, 249, 217, 230, 144]) })], threshold: 1, vesting_duration: 17102566571104181794, vesting_start: 0 }), balance: TokenAmount(219517841402426701068064509197116201799893053391202422345016930037028485502523110328182626063084151690909564295277313.991025903641489249) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Delegated(DelegatedAddress { namespace: 10, length: 20, buffer: [179, 190, 148, 15, 154, 129, 0, 136, 107, 155, 18, 194, 6, 67, 39, 17, 166, 104, 203, 247, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0] }) }) }), balance: TokenAmount(24504241214570137024925845315545583050415672186902331628271827919529368995351336875591376877933523.875463162164835287) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Secp256k1([92, 5, 208, 34, 219, 206, 47, 143, 32, 184, 27, 45, 150, 95, 204, 138, 230, 208, 86, 235]) }) }), balance: TokenAmount(41855804968213567224547853478906320725054875457247406540771499545716837934567817284890561672488119458109166910841919797858872745645489779490809650234062883597999517971618161081243073047294805204605864371183315246318231940454337050599284218596517288593326458012817130715943156143381.908498439530633444) }, Actor { meta: Account(Account { owner: SignerAddr(Address { payload: Secp256k1([121, 42, 181, 92, 162, 16, 136, 148, 115, 30, 219, 155, 235, 11, 35, 127, 199, 98, 235, 46]) }) }), balance: TokenAmount(0.0) }], ipc: None }